scripts/pam-test.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+set -e
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+bb_print_env
```

Comment:
already done in `rpm-install.sh`, probably not needed? No strong opinion, this might be helpful to not have to open previous step to check env.

Reply:
I put it there for user convenience if the step is failed and want to have a quick look on the environment setup.